examples/minio/05-ark-backupstoragelocation.yaml

```diff
@@ -26,5 +26,7 @@ spec:
     region: minio
     s3ForcePathStyle: "true"
     s3Url: http://minio.heptio-ark.svc:9000
+    # OR get minio URL (including http/s) per documentation (comment out previous line)
+    # s3Url: NODE_URL_OR_IP:NODE_PORT
```

Comment:
let's change this line to be `publicUrl: ...` -- they'll leave the `s3Url` as-is, and only change `publicUrl`